javascript/boilerplate-text/boilerplate-zh.js

```diff
@@ -0,0 +1,136 @@
+// translate the text between quotes
```

Comment:
Actually, i guess that this is a new file has a -zh filename.  We should instead be using the file at https://w3c.github.io/i18n-drafts/javascript/boilerplate-text/boilerplate-zh-hans.js